examples/oauth-express-receiver/app.js

```diff
@@ -1,5 +1,10 @@
 const { App, ExpressReceiver, LogLevel, FileInstallationStore } = require('@slack/bolt');
 
+/**
+ * @type {import("@slack/bolt/dist/receivers/ParamsIncomingMessage").ParamsIncomingMessage};
+ */
+const _example = null;
```

Comment:
TBH I think `ParamsIncomingMessage should be exported at the top-level of `@slack/bolt` since it describes the "req" parameter of receivers, but that's unchanged in this PR.

**However** I'm super curious to know if exporting it from the top-level would fix the current `typescript` error here 🤔